golang/cmd/mqtt-kafka-bridge/mqtt.go

```diff
@@ -21,23 +21,40 @@ func newTLSConfig() *tls.Config {
 	// Alternatively, manually add CA certificates to
 	// default openssl CA bundle.
 	certpool := x509.NewCertPool()
-	pemCerts, err := os.ReadFile("/SSL_certs/ca.crt")
+	pemCerts, err := os.ReadFile("/SSL_certs/mqtt/ca.crt")
 	if err == nil {
-		certpool.AppendCertsFromPEM(pemCerts)
+		ok := certpool.AppendCertsFromPEM(pemCerts)
+		if !ok {
+			zap.S().Errorf("Failed to parse root certificate")
+		}
+	} else {
+		zap.S().Errorf("Error reading CA certificate: %s", err)
 	}
 
+	zap.S().Debugf("CA cert: %s", pemCerts)
+
 	// Import client certificate/key pair
-	cert, err := tls.LoadX509KeyPair("/SSL_certs/tls.crt", "/SSL_certs/tls.key")
+	cert, err := tls.LoadX509KeyPair("/SSL_certs/mqtt/tls.crt", "/SSL_certs/mqtt/tls.key")
 	if err != nil {
-		panic(err)
+		// Read /SSL_certs/mqtt/tls.crt
+		var file []byte
+		file, err = os.ReadFile("/SSL_certs/mqtt/tls.crt")
+		if err != nil {
+			zap.S().Errorf("Error reading client certificate: %s", err)
+		}
+		zap.S().Fatalf("Error reading client certificate: %s (File: %s)", err, file)
 	}
 
+	zap.S().Debugf("Client cert: %v", cert)
+
 	// Just to print out the client certificate..
 	cert.Leaf, err = x509.ParseCertificate(cert.Certificate[0])
 	if err != nil {
-		panic(err)
+		zap.S().Fatalf("Error parsing client certificate: %s", err)
 	}
 
+	skipVerify := os.Getenv("INSECURE_SKIP_VERIFY") == "true"
```

Comment:
variable is not documented yet, also not consistent with mqtt-bridge (there is no variable yet)

Reply:
Added documentation in values.yaml